x-pack/dockerlogbeat/readme.md

```diff
@@ -12,6 +12,22 @@ To build and install, just run `mage Package`. The build process happens entire
 `docker run --log-driver=elastic-logging-plugin:8.0.0 --log-opt output.elasticsearch.hosts="172.18.0.2:9200" --log-opt output.elasticsearch.index="dockerbeat-test" -it debian:jessie /bin/bash`
 
 
+## Config Options
+
+| Option               | Description                                                                                                                       |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `hosts`              | The elasticsearch endpoint to connect to. This option is required. Multiple endpoints may be specified and delimited with a `,`   |
```

Comment:
A couple of thoughts:

* If we duplicate descriptions that are also covered in the docs, we should at least be consistent and use the same wording.

* I think it's OK for these descriptions to be shorter and contain less detail than the docs, but we should at least point to the docs for more detail.

Reply:
Yah. I want some documentation to be on the readme since that's what people will see when they land on the github page. Might just be better to have an example or two and then link to the asciidocs?